### ISSUE TITLE
JPverb causes Server crash with 3.8.0 (Windows VS 64-bit)

### ISSUE THREAD

**Issue by totalgee** [supercollider/sc3-plugins#174]
As mentioned in #133 , JPverb still crashes the Server with 3.8.0, when using the Visual Studio-compiled 64-bit version on Windows. There is no workaround that I know of. @LFSaw asked me to open a new issue for it.

Repro steps:

```supercollider
{ JPverb.ar(Impulse.ar(1, 0, 0.5)) }.play;
```

I believe the mentioned workaround (increase memSize: `s.options.memSize = 2**18;`) works on the 3.8.0 x86 (32-bit) version, and it also works fine on 3.9-beta1. However, it does not work for me with the 64-bit "official" release of 3.8, not even with much larger memSize values (512MB or more).

**Comment by dvzrv** [comment]
This also crashes on Linux with SuperCollider 3.9.3, but only after 3 instances of above example.
